NOTICE

```diff
@@ -0,0 +1,5 @@
+Apache Podling-Texera
```

Comment:
Do we also need a NOTICE-binary file?

Reply:
No, since we don't have a binary release plan yet.

Reply:
We just made a v1.0.0 release, which includes k8s images of binaries, right?

Reply:
I don't think k8s image is actually a binary. As long as we include the NOTICE file in the image, we are fine. The main reason the other projects include a binary version of NOTICE is that they cannot or they don't want to compile the regular version of NOTICE file into the binary release.

Reply:
If it is easy and no harm to include a NOTICE-binary file, let's just add it?

Reply:
Let's not include unnecessary files. 

> As a result, the LICENSE and NOTICE files for a binary distribution may differ from those in the source distribution it was built from.

We don't need to build a binary release that requires a different version of the NOTICE file.

https://infra.apache.org/licensing-howto.html

Reply:
ok